src/contracts/gauntlet/AaveV3AvaRiskParameterUpdate.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import '../common/StewardBase.sol';
+import {AaveV3Avalanche} from 'aave-address-book/AaveV3Avalanche.sol';
+
+uint256 constant NUM_UPDATES = 5;
+
+struct Updates {
```

Comment:
Of course, it works and is correct, but I wonder why you decided to declare this way instead of returning in `_getUpdates()` a `ParameterSet[] memory`